.github/workflows/docs.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: master
+          ref: vb/marimo
```

Comment:
This change is only to build the `wasm/` folder using this branch, and should go away before we do an actual merge.